### PR TITLE
DCO: Update Linux Foundation address (to Letterman Drive)

### PR DESCRIPTION
**Pull request by wking**
Generated with:

```console
$ curl -s https://developercertificate.org >DCO
```

after which I stripped the `<pre>` tags and everything outside of them.

### CODE REVIEW

File: DCO [hunk 1]
```diff
@@ -2,8 +2,9 @@ Developer Certificate of Origin
 Version 1.1
 
 Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
-660 York Street, Suite 102,
-San Francisco, CA 94110 USA
+1 Letterman Drive
+Suite D4700
+San Francisco, CA, 94129
```
Comment (squat) [inline DCO]: No comma is needed between the state and the ZIP

File: DCO [hunk 2]
```diff
@@ -2,8 +2,9 @@ Developer Certificate of Origin
 Version 1.1
 
 Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
-660 York Street, Suite 102,
-San Francisco, CA 94110 USA
+1 Letterman Drive
+Suite D4700
```
Comment (squat) [inline DCO]: Does this line need the building number too? `Building D, Suite D4700`? as in https://www.linuxfoundation.org/about/contact/

File: DCO (hunk 1 above)
Comment (wking) [reply DCO]: > No comma is needed between the state and the ZIP

This is what came out of [`curl`ing https://developercertificate.org][1].  With the DCO distributed under [verbatim-copies-only][2] terms, I don't think it's worth tuning their content.  Maybe submit these suggestions upstream to the LF?

[1]: https://github.com/openshift/installer/pull/26#issue-199841667
[2]: https://github.com/openshift/installer/blob/9811c32af5bffebf42237cb94068f008655a307c/DCO#L8-L9

Comment (squat) [reply DCO]: sorry I missed that. this in indeed the content of the hosted document :/

**Comment by wking** [comment]
This PR had the same [operator Instantiate the template e2e-aws into the operator namespace and wait for any pods to complete][1] error as [seen][2] in #29.  It also [had][1]:

```
[Conformance][Area:Networking][Feature:Router] The HAProxy router should set Forwarded headers appropriately [Suite:openshift/conformance/parallel] 0.80s
go run hack/e2e.go -v -test --test_args='--ginkgo.focus=\[Conformance\]\[Area\:Networking\]\[Feature\:Router\]\sThe\sHAProxy\srouter\sshould\sset\sForwarded\sheaders\sappropriately\s\[Suite\:openshift\/conformance\/parallel\]$'
/tmp/openshift/build-rpms/rpm/BUILD/origin-3.11.0/_output/local/go/src/github.com/openshift/origin/test/extended/util/cli.go:86
Jul  6 23:57:38.542: the server could not find the requested resource (post users.user.openshift.io)
/tmp/openshift/build-rpms/rpm/BUILD/origin-3.11.0/_output/local/go/src/github.com/openshift/origin/test/extended/util/cli.go:655
```

[1]: https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_installer/26/pull-ci-origin-installer-e2e-aws/88/
[2]: https://github.com/openshift/installer/pull/29#issuecomment-403172107

**Pull request merged**